### PR TITLE
93236 mut scaffold

### PR DESCRIPTION
**Pull request by adunndevster**

### CODE REVIEW

File: docs-metadata/src/controllers/apply-controller.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	env, 
+	window, 
+	workspace, 
+	ExtensionContext, 
+	OpenDialogOptions,
+	Uri} from 'vscode';
+
+export async function showApplyMetadataMessage(folderPath:string)
+{
+	window.showInformationMessage(`Apply MUT file ${folderPath}?`,
```
Comment (bharney-msft) [inline docs-metadata/src/controllers/apply-controller.ts]: we should probably spell out the entire word instead of using MUT

Comment (adunndevster) [reply docs-metadata/src/controllers/apply-controller.ts]: Hmmm... didn't think about it before by I agree. This message was in the spec specifically.

File: docs-metadata/package.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	"activationEvents": [
+		"onCommand:extension.extract",
+		"onCommand:extension.apply"
+	],
+	"main": "./out/extension.js",
+	"contributes": {
+		"commands": [
+			{
+				"command": "extension.extract",
+				"title": "Docs Metadata: Extract"
```
Comment (jamarw) [inline docs-metadata/package.json]: @adunndevster Everything looks great!  One question: do you think the title should be tweaked to read **Docs: Metadata extract** and **Docs: Metadata apply** to remain consistent with the other commands in the command palette?  Not a big deal at all and maybe it's different by design.  

image

Comment (adunndevster) [reply docs-metadata/package.json]: @meganbradley - What do you think?

Comment (adunndevster) [reply docs-metadata/package.json]: I'll change it. Easy enough to change back if needed :).

File: docs-metadata/src/extension.ts [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+                const downloader = new ExtensionDownloader(logger, extension.packageJSON);
+                return downloader.installRuntimeDependencies();
+            } else {
+                return true;
+            }
+        });
+}
+
+function platformIsSupported(logger: Logger): boolean {
+	var getos = require('getos')
```
Comment (jamarw) [inline docs-metadata/src/extension.ts]: Hi @adunndevster , there's a native node module may work here to reduce some code and drop a dependency. https://nodejs.org/dist/latest-v10.x/docs/api/os.html#os_os_platform

File: docs-metadata/.vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    "files.exclude": {
+        "out": false // set this to true to hide the "out" folder with the compiled JS files
+    },
+    "search.exclude": {
+        "out": true // set this to false to include "out" folder in search results
+    },
+    // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
+    "typescript.tsc.autoDetect": "off",
+    "cSpell.words": [
+        "Synchrounous"
```
Comment (nschonni) [inline docs-metadata/.vscode/settings.json]: Doesn't look like this is needed now

File: docs-metadata/.gitignore [hunk 5]
```diff
@@ -0,0 +1,2 @@
+node_modules/*
+out/*
```
[the remark this answers is not in the document]
Comment (adunndevster) [reply docs-metadata/.gitignore]: @jamarw thank you for fixing!

**Pull request merged**